samples/videoDecode/videodecode.cpp

```diff
@@ -329,17 +329,17 @@ int main(int argc, char **argv) {
                 std::cout << std::setfill('0') << std::setw(2) << std::hex << static_cast<int>(digest[i]);
             }
             std::cout << std::endl;
-
             if (b_md5_check) {
                 char ref_md5_string[33], c2[2];
                 uint8_t ref_md5[16];
-                std::string str;
+                std::string str(2,0);
```

Comment:
please change the name of the string to different name as 'str' is a keyword in std::string lib.
std::string x: is good enough

Reply:
Done.